bigtop-packages/src/deb/hbase/rules

```diff
@@ -50,15 +53,15 @@ override_dh_auto_install:
 	ln -f -s ${hadoop_home}/client/hadoop-annotations.jar debian/tmp/usr/lib/${hbase_pkg_name}/lib/
 	ln -f -s ${hadoop_home}/client/hadoop-auth.jar debian/tmp/usr/lib/${hbase_pkg_name}/lib/
 	ln -f -s ${hadoop_home}/client/hadoop-common.jar debian/tmp/usr/lib/${hbase_pkg_name}/lib/
-	ln -f -s ${hadoop_home}/client/hadoop-hdfs.jar debian/tmp/usr/lib/${hbase_pkg_name}/lib/
-	ln -f -s ${hadoop_home}/client/hadoop-mapreduce-client-app.jar debian/tmp/usr/lib/${hbase_pkg_name}/lib/
+	ln -f -s ${hadoop_hdfs_path}/hadoop-hdfs.jar debian/tmp/usr/lib/${hbase_pkg_name}/lib/
+	ln -f -s ${hadoop_mapreduce_path}/hadoop-mapreduce-client-app.jar debian/tmp/usr/lib/${hbase_pkg_name}/lib/
 	ln -f -s ${hadoop_home}/client/hadoop-mapreduce-client-common.jar debian/tmp/usr/lib/${hbase_pkg_name}/lib/
 	ln -f -s ${hadoop_home}/client/hadoop-mapreduce-client-core.jar debian/tmp/usr/lib/${hbase_pkg_name}/lib/
 	ln -f -s ${hadoop_home}/client/hadoop-mapreduce-client-jobclient.jar debian/tmp/usr/lib/${hbase_pkg_name}/lib/
-	ln -f -s ${hadoop_home}/client/hadoop-mapreduce-client-shuffle.jar debian/tmp/usr/lib/${hbase_pkg_name}/lib/
+	ln -f -s ${hadoop_mapreduce_path}/hadoop-mapreduce-client-shuffle.jar debian/tmp/usr/lib/${hbase_pkg_name}/lib/
 	ln -f -s ${hadoop_home}/client/hadoop-yarn-api.jar debian/tmp/usr/lib/${hbase_pkg_name}/lib/
 	ln -f -s ${hadoop_home}/client/hadoop-yarn-client.jar debian/tmp/usr/lib/${hbase_pkg_name}/lib/
-	ln -f -s ${hadoop_home}/client/hadoop-yarn-common.jar debian/tmp/usr/lib/${hbase_pkg_name}/lib/
+	ln -f -s ${hadoop_yarn_path}/hadoop-yarn-common.jar debian/tmp/usr/lib/${hbase_pkg_name}/lib/
 	ln -f -s ${hadoop_home}/client/hadoop-yarn-server-common.jar debian/tmp/usr/lib/${hbase_pkg_name}/lib/
```

Comment:
hadoop-yarn-server-common.jar needs the same fix? I tested the patch like following. @guluo2016 

```
[root@ec68ac35bd71 /]# for f in $(find /usr/lib/hbase/lib -type l) ; do test -e "$f" || echo "$f does not exist." ; done
/usr/lib/hbase/lib/hadoop-yarn-server-common.jar does not exist.
```